src/Components/Card.jsx

```diff
@@ -0,0 +1,40 @@
+import "./Styles/Card.css";
+
+import React from "react";
```

Comment:
From React version 17, this import is not required.

Reply:
Updated it in the latest commit